woo-razorpay.php

```diff
@@ -588,13 +588,18 @@ private function getRedirectUrl($orderId)
         protected function getRazorpayPaymentParams($orderId)
         {
             $getWebhookFlag =  get_option('webhook_enable_flag');
-
+            $time = time();
            if (!empty($getWebhookFlag))
            {
                 if ($getWebhookFlag + 86400 < time())
                 {
                     $this->autoEnableWebhook(); 
                 }
+           }
+           else
+           {
+                add_option('webhook_enable_flag', $time);
```

Comment:
@jampulanaveen `add_option` is already called in `autoEnableWebhook` function, why again it's required here?
`if (empty($getWebhookFlag))
            {
                add_option('webhook_enable_flag', $time);
            }
            else
            {
                update_option('webhook_enable_flag', $time);
            }`
I don't think this check is required. Instead of that if we use `update_option` then it will add the flag if does not exist. Refer this [link](https://developer.wordpress.org/reference/functions/update_option/)

Reply:
ok i will update it